java/src/jmri/jmrix/loconet/LnConstants.java

```diff
@@ -393,7 +393,8 @@ public final static String DEC_MODE(int s) { // encode decoder type as a string
     public final static int OPC_EXP_SEND_SUB_CODE_MASK_SPEED = 0b11110000;
     public final static int OPC_EXP_SEND_SUB_CODE_MASK_FUNCTION = 0b11111000;
     public final static int OPC_EXP_SEND_FUNCTION_OR_SPEED_AND_DIR = 0xd5;
-    public final static int OPC_EXP_SEND_SPEED_AND_DIR_MASK = 0b00010000;
+    public final static int OPC_EXP_SEND_SPEED_AND_DIR_MASK = 0b00000000;
```

Comment:
What's the purpose of this constant? `MASK` usually stands for masking, like in this line:
`} else if ((l.getElement(1) & LnConstants.OPC_EXP_SEND_SUB_CODE_MASK_FUNCTION) == LnConstants.OPC_EXP_SEND_FUNCTION_GROUP_F0F6_MASK) {`

But this mask is 0, so it masks everything. Is it right?

Reply:
Renamed